packages/schematics/migrations/update-6-0-0/update-6-0-0.ts

```diff
@@ -289,14 +290,22 @@ function deleteUnneededFiles(host: Tree) {
   return host;
 }
 
-function patchLibIndexFiles(host: Tree) {
+function patchLibIndexFiles(host: Tree, context: SchematicContext) {
   const angularJson = readJsonInTree(host, 'angular.json');
 
   Object.values(angularJson.projects).forEach((p: any) => {
     if (p.projectType === 'library') {
       fs.renameSync(p.sourceRoot, `${p.root}/lib`);
       fs.mkdirSync(p.sourceRoot);
       fs.renameSync(`${p.root}/lib`, `${p.sourceRoot}/lib`);
+      const npmScope = readJsonInTree(host, 'nx.json').npmScope;
+      host = updateJsonInTree('tsconfig.json', json => {
```

Comment:
either rewrite it with read and override, or rewrite the whole patchLibIndexFiles as a chain of rules.  

This looks a bit convoluted for updating a single field.